pkg/controllers/management/auth/project_cluster/common.go

```diff
@@ -0,0 +1,73 @@
+package project_cluster
+
+import (
+	"github.com/pkg/errors"
+	"github.com/rancher/norman/condition"
+	apisv3 "github.com/rancher/rancher/pkg/apis/management.cattle.io/v3"
+	corev1 "github.com/rancher/rancher/pkg/generated/norman/core/v1"
+	"github.com/sirupsen/logrus"
+	v12 "k8s.io/api/core/v1"
+	apierrors "k8s.io/apimachinery/pkg/api/errors"
+	"k8s.io/apimachinery/pkg/api/meta"
+	v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+)
+
+const (
+	CreatorIDAnn                  = "field.cattle.io/creatorId"
```

Comment:
Looks like we could standardise the naming?

These are all used as annotation keys?